### ISSUE TITLE
Developer to launch AI art app (open source) 

### ISSUE THREAD

**Issue by elenajoness** [stablecog/stablecog#18]
Hi Sir,
Hope you are doing well,

✅ You said that "I have an open source system that I'd like to replicate in my domain for creating AI art images. 
...." and we can do it with very high quality and fast, you can check our strong portfolio during meeting to make sure.

📩 Mind if I send over a few times to quick chat?

Thanks, 
- Elena.

**Comment by yekta** [comment]
I’m not sure what do you mean exactly since we are already running a service and this is our repository. Maybe you intended for someone else? Regardless of that, this is not the use case for issues. If you have a question related to our service, you can just email us.

**Comment by elenajoness** [comment]
Sure, Sorry for that and May I get you your email and send the details.


[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
06/03/23,
10:36:29 AM

**Comment by yekta** [comment]
I don’t think you understand what I said. To repeat, we are the creators of Stablecog. It’s an open-source project we are also hosting at https://stablecog.com.

We do not require a service to “replicate it in our own domain”. We built the thing :) We know how to host it, we are hosting it.

I suspect someone emailed you asking to clone Stablecog on their own domain and you came here opening an issue. This is not the place, you can talk to them via their email.
